fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/database/entities/RedirectUrlsEntity.java

```diff
@@ -2,20 +2,25 @@
 
 import lombok.Data;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
 
 @Data
 @Entity
 public class RedirectUrlsEntity {
     @Id
-    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "redirect_urls_generator")
+    @SequenceGenerator(name = "redirect_urls_generator", sequenceName = "redirect_urls_id_seq", allocationSize = 1)
```

Comment:
Using allocationSize 1 is a bad idea from a performance perspective as it is an entity-per-request class. Use 50 as sane default. See below

Reply:
done